src/modules/ekf2/EKF2.cpp

```diff
@@ -1742,7 +1745,7 @@ int EKF2::task_spawn(int argc, char *argv[])
 
 		while ((multi_instances_allocated < multi_instances)
 		       && (vehicle_status_sub.get().arming_state != vehicle_status_s::ARMING_STATE_ARMED)
-		       && (hrt_elapsed_time(&time_started) < 30_s)) {
+		       && ((hrt_elapsed_time(&time_started) < 30_s || hitl_mode == 1))) {
```

Comment:
Actually, we can grab this from vehicle_status. I'll update it quickly.

Reply:
Ah sure, even better! Thx.